### PR TITLE
Feature/tag source

### PR DESCRIPTION
**Pull request by jthompkins**
Added external-secrets entry for tag-source github workflow step

### CODE REVIEW

File: templates/external-secrets.yaml [hunk 1]
```diff
@@ -115,6 +115,12 @@ spec:
                 {{`{{ .registry2Host }}`}}:
                   username: {{`{{ .registry2User }}`}}
                   password: {{`{{ .registry2Password }}`}}
+            tag-source:
+            - implementer: Git
+              config:
+                git-username: {{`{{ .gitUsername }}`}}
+                git-user-name: {{`{{ .gitUsername }}`}}
```
Comment (dwinchell) [inline templates/external-secrets.yaml:122]: Do we actually still need both of these?

**Review by dwinchell — CHANGES_REQUESTED** [review]

File: templates/external-secrets.yaml (hunk 1 above)
Comment (aarongreen85) [reply templates/external-secrets.yaml:122]: @jthompkins we can likely remove line 122. what do you think?

Comment (jthompkins) [reply templates/external-secrets.yaml]: testing removal now

Comment (jthompkins) [reply templates/external-secrets.yaml]: yeah it still works without user-name

**Pull request merged**